File: beecrowd/C++20/1036.cpp
#include <iostream>
#include <math.h>
using namespace std;
 
int main() {
    double a, b, c, delta;

    cin >> a >> b >> c;

    delta = (b*b) - (4*a*c);

    if (delta < 0 or a == 0) {
        cout << "Impossivel calcular" << "\n";
    } else {
        cout.precision(5);
        cout << fixed << "R1 = " << (-b + sqrt(delta)) / (2*a) << "\n";
        cout << fixed << "R2 = " << (-b - sqrt(delta)) / (2*a) << "\n";
    }
    return 0;
}

File: beecrowd/python3-9/1036.py
# -*- coding: utf-8 -*-

import math

A, B, C = map(float, input().split())

delta = B**2 - 4*A*C

if delta < 0:
    print('Impossivel calcular')
elif A == 0:
    print('Impossivel calcular')
else:
    root1 = (-B + math.sqrt(delta)) / (2*A)
    root2 = (-B - math.sqrt(delta)) / (2*A)
    print(f'R1 = {root1:.5f}')
    print(f'R2 = {root2:.5f}')